### DISCUSSION TITLE
German Translation dilemma - formal or informal?

### DISCUSSION THREAD

**Discussion by phil-flip** [snipe/snipe-it#11312] [Polls]
For the German translation, should it be formal or informal?
In German, like with many languages, you can express in a wide variety. Because Germany is quite the bureaucracy, it was more common to be formal to everyone you don't know personally. For example, the way you would speak in the US to an officer or a jury, with 'Sir' and whatnot. However, the more IT started to sprout, this standard seem to slowly shift towards the informal German. For Example, you would talk to a close friend or in the family. This is now quite common to speak the informal now in Germany, even to people you don't know.

The current problem is, that both types are present in the project, causing inconsistencies from menu to menu. This is because of the transition which is happening at the moment. So, before this gets out of hand and I start translating wrong. I want to settle this, and maybe it helps also other languages, which have a similar issue.
This should be decided by the project owners. But I still set up a poll to see what everyone thinks of this translation issue.

**Comment by phil-flip** [comment]
I recommend sticking to informal, as the informal expression slowly gets dated and less used overall in applications like this.

**Comment by snipe** [comment]
@FlipperLP We've just started a 4-day weekend here in the states, but I've pinged @bzeus (she works here at Grokability and is from Germany) to weigh in on Tuesday.

**Comment by snipe** [comment]
Okay - we now have two

https://crowdin.com/project/snipe-it/de
https://crowdin.com/project/snipe-it/de-i

We'll work on getting them consistent with the descriptions.

**Reply by phil-flip** [reply]
So I'm guessing de-i is the informal version?

**Reply by phil-flip** [reply]
Would it be possible to copy all the formal strings over to informal? That would make it easier, since they are quite similar, and it helps with consistency between the 2 versions

**Reply by phil-flip** [reply]
ah nvm.. Might not be necessary as it already pulls it from the other version

**Reply by snipe** [reply]
It's based off of existing German. @bzeus and others will work on making them consistent

**Comment by snipe** [comment]
(We can't default to informal, since many enterprise users - especially in banking and healthcare - prefer more formal.)

**Reply by phil-flip** [reply]
I guess you can just add it as an extra language in the dropdown menu, if it allows you to do so.

**Reply by snipe** [reply]
Yes, that's how we'd do it. Instead of just German, there will be `German` and `German - Informal`
